scripts/completion/dvc.bash

```diff
@@ -82,13 +86,21 @@ _dvc () {
       *)  COMPREPLY=($(compgen -W "$_dvc_commands" -- "$word")) ;;
     esac
   elif [ "${COMP_CWORD}" -eq 2 ]; then
-    local options_list="_dvc_${COMP_WORDS[1]}"
+    if [[ ${COMP_WORDS[1]:0:1} == "-" ]]; then
```

Comment:
No need for this if block, the fact that you are replacing `-` with `_` will simply evaluate the empty variable, so it is going to simply work.